index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+        </div>
+    <div id="img-div">
+      <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+    <p id="img-caption"> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger. </p>
+    </div>
+    <div id="tribute-info">
+
+    </div>
+    <p>If you have time, you should read more about this inredible human being on his 
+    <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry</a></p>
+    </div>
```

Comment:
Please use footer to make your code semantic.

Reply:
- [ ] Good one here also. Nice feedback but it could be very explicit to add why they should use the `footer` tag.
- Your code author should know what the Htm5 semantic tags are and why exactly to use the `footer tag`. I think adding an external link to the htm5 semantic tag will make your review better and increase the chance of the code author understanding your review better.